ovos_config/meta.py

```diff
@@ -178,6 +173,7 @@ def set_config_filename(file_name, core_folder=None):
     if core_folder:
         set_xdg_base(core_folder)
     LOG.info(f"config filename set to: '{file_name}'")
+    os.environ["OVOS_CONFIG_FILENAME"] = file_name
```

Comment:
Will need to look into side effects here since `os.env` could affect code outside of this module?